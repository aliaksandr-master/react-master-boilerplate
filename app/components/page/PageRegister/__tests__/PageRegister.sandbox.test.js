/* eslint-env browser, jest */
/* eslint-disable react/jsx-no-bind */

import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandboxPageRegister from '../PageRegister.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandboxPageRegister).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const PageRegisterSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <PageRegisterSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
