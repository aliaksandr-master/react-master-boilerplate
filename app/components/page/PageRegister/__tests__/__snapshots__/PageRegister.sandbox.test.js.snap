// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div>
  <div
    className="PageRegister-root-0-1-1"
  >
    <div
      className="PageRegister-paper-0-1-2"
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        }
      }
    >
      <div>
        <div
          className="PageRegister-h1-0-1-7"
        >
          Tests
        </div>
        <div
          className="PageRegister-inputWr-0-1-4"
        >
          <div
            className={undefined}
            style={
              Object {
                "backgroundColor": "transparent",
                "cursor": "auto",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "height": 48,
                "lineHeight": "24px",
                "muiPrepared": true,
                "position": "relative",
                "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "width": 256,
              }
            }
          >
            <input
              disabled={false}
              id="f1485955293000"
              name="username"
              onBlur={[Function]}
              onChange={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              placeholder="Nick Name"
              size="usymbol-0$FieldInput-sizeLarge"
              style={
                Object {
                  "WebkitOpacity": 1,
                  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                  "appearance": "textfield",
                  "backgroundColor": "rgba(0,0,0,0)",
                  "border": "none",
                  "color": "rgba(0, 0, 0, 0.87)",
                  "cursor": "inherit",
                  "font": "inherit",
                  "height": "100%",
                  "muiPrepared": true,
                  "outline": "none",
                  "padding": 0,
                  "position": "relative",
                  "width": "100%",
                }
              }
              type="text"
              value=""
            />
            <div>
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 1,
                    "borderColor": "#e0e0e0",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "width": "100%",
                  }
                }
              />
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 2,
                    "borderColor": "#00bcd4",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "transform": "scaleX(0)",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "width": "100%",
                  }
                }
              />
            </div>
            
          </div>
        </div>
        <div
          className="PageRegister-inputWr-0-1-4"
        >
          <div
            className="PageRegister__formInput"
            style={
              Object {
                "backgroundColor": "transparent",
                "cursor": "auto",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "height": 48,
                "lineHeight": "24px",
                "muiPrepared": true,
                "position": "relative",
                "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "width": 256,
              }
            }
          >
            <input
              disabled={false}
              id="f1485955293000"
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              placeholder="Email"
              size="usymbol-0$FieldInput-sizeLarge"
              style={
                Object {
                  "WebkitOpacity": 1,
                  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                  "appearance": "textfield",
                  "backgroundColor": "rgba(0,0,0,0)",
                  "border": "none",
                  "color": "rgba(0, 0, 0, 0.87)",
                  "cursor": "inherit",
                  "font": "inherit",
                  "height": "100%",
                  "muiPrepared": true,
                  "outline": "none",
                  "padding": 0,
                  "position": "relative",
                  "width": "100%",
                }
              }
              type="email"
              value=""
            />
            <div>
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 1,
                    "borderColor": "#e0e0e0",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "width": "100%",
                  }
                }
              />
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 2,
                    "borderColor": "#00bcd4",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "transform": "scaleX(0)",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "width": "100%",
                  }
                }
              />
            </div>
            
          </div>
        </div>
        <div
          className="PageRegister-inputWr-0-1-4"
        >
          <div
            className="PageRegister__formInput"
            style={
              Object {
                "backgroundColor": "transparent",
                "cursor": "auto",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "height": 48,
                "lineHeight": "24px",
                "muiPrepared": true,
                "position": "relative",
                "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "width": 256,
              }
            }
          >
            <input
              disabled={false}
              id="f1485955293000"
              name="password"
              onBlur={[Function]}
              onChange={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              placeholder="Password"
              size="usymbol-0$FieldInput-sizeLarge"
              style={
                Object {
                  "WebkitOpacity": 1,
                  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                  "appearance": "textfield",
                  "backgroundColor": "rgba(0,0,0,0)",
                  "border": "none",
                  "color": "rgba(0, 0, 0, 0.87)",
                  "cursor": "inherit",
                  "font": "inherit",
                  "height": "100%",
                  "muiPrepared": true,
                  "outline": "none",
                  "padding": 0,
                  "position": "relative",
                  "width": "100%",
                }
              }
              type="password"
              value=""
            />
            <div>
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 1,
                    "borderColor": "#e0e0e0",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "width": "100%",
                  }
                }
              />
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 2,
                    "borderColor": "#00bcd4",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "transform": "scaleX(0)",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "width": "100%",
                  }
                }
              />
            </div>
            
          </div>
        </div>
        <div
          className="PageRegister-inputWr-0-1-4"
        >
          <div
            className="PageRegister__formInput"
            style={
              Object {
                "backgroundColor": "transparent",
                "cursor": "auto",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "height": 48,
                "lineHeight": "24px",
                "muiPrepared": true,
                "position": "relative",
                "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "width": 256,
              }
            }
          >
            <input
              disabled={false}
              id="f1485955293000"
              name="confirm"
              onBlur={[Function]}
              onChange={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              placeholder="Confirm Password"
              size="usymbol-0$FieldInput-sizeLarge"
              style={
                Object {
                  "WebkitOpacity": 1,
                  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                  "appearance": "textfield",
                  "backgroundColor": "rgba(0,0,0,0)",
                  "border": "none",
                  "color": "rgba(0, 0, 0, 0.87)",
                  "cursor": "inherit",
                  "font": "inherit",
                  "height": "100%",
                  "muiPrepared": true,
                  "outline": "none",
                  "padding": 0,
                  "position": "relative",
                  "width": "100%",
                }
              }
              type="password"
              value=""
            />
            <div>
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 1,
                    "borderColor": "#e0e0e0",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "width": "100%",
                  }
                }
              />
              <hr
                aria-hidden="true"
                style={
                  Object {
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": 2,
                    "borderColor": "#00bcd4",
                    "borderLeft": "none",
                    "borderRight": "none",
                    "borderTop": "none",
                    "bottom": 8,
                    "boxSizing": "content-box",
                    "margin": 0,
                    "muiPrepared": true,
                    "position": "absolute",
                    "transform": "scaleX(0)",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "width": "100%",
                  }
                }
              />
            </div>
            
          </div>
        </div>
        <noscript />
        <div
          className="PageRegister-btnZone-0-1-6"
        >
          <a
            className="PageRegister-loginBtn-0-1-3"
            disabled={false}
            href="/login"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": null,
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              Log in
            </div>
          </a>
          <div
            className="PageRegister-submitBtn-0-1-5"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": 2,
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "minWidth": 88,
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              }
            }
          >
            <button
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                  "backgroundColor": "#00bcd4",
                  "border": 10,
                  "borderRadius": 2,
                  "boxSizing": "border-box",
                  "cursor": "pointer",
                  "display": "inline-block",
                  "fontFamily": "Roboto, sans-serif",
                  "fontSize": "inherit",
                  "fontWeight": "inherit",
                  "height": 36,
                  "lineHeight": "36px",
                  "margin": 0,
                  "muiPrepared": true,
                  "outline": "none",
                  "padding": 0,
                  "position": "relative",
                  "textAlign": "center",
                  "textDecoration": "none",
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  "verticalAlign": null,
                  "width": "100%",
                }
              }
              tabIndex={0}
              type="submit"
            >
              <div
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
              >
                <div
                  style={
                    Object {
                      "backgroundColor": false,
                      "borderRadius": 2,
                      "height": 36,
                      "muiPrepared": true,
                      "top": 0,
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    }
                  }
                >
                  Register
                </div>
              </div>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
